maxim/logger/openai/completions.py

```
@@ -6,10 +6,54 @@
 from typing_extensions import override
 
 from ...scribe import scribe
-from ..logger import Generation, GenerationConfig, Logger, Trace, TraceConfig
+from ..logger import Generation, Logger, Trace
 from .utils import OpenAIUtils
 
 
+class StreamWrapper:
+    def __init__(self, stream, generation, trace, is_local_trace):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)